examples/create_remote_by_host.py

```diff
@@ -0,0 +1,140 @@
+"""Get all hosts in specified subnets and create an agentless connection if not supervised """
+
+import os
+import ipaddress
+import argparse
+from configparser import ConfigParser
+from cbw_api_toolbox.cbw_api import CBWApi
+
+
+def connect_api():
+    '''Connect to the API and test connection'''
+    conf = ConfigParser()
+    conf.read(os.path.join(os.path.abspath(
+        os.path.dirname(__file__)), '..', 'api.conf'))
+    global CLIENT  # pylint: disable=global-variable-undefined
+    CLIENT = CBWApi(conf.get('cyberwatch', 'url'), conf.get(
+        'cyberwatch', 'api_key'), conf.get('cyberwatch', 'secret_key'))
+
+    CLIENT.ping()
+
+
+# Change these variables to NOT use the default credentials for SSH set up in Cyberwatch
+SSH_LOGIN = ""
+SSH_PASSWORD = ""
+
+NEW = []
+EXISTING = []
+ERRORS = []
+
+
+def create_agentless(host, parsed_args):
+    """Create an agentless connection if not in read-only"""
+    info = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
+            "address": host.target,
+            "port": "22",
+            "node_id": host.node_id,
+            "login": SSH_LOGIN,
+            "password": SSH_PASSWORD
+            }
+    if parsed_args.i is True:
+        create = CLIENT.create_remote_access(info)
+        if create is False:
+            ERRORS.append(host)
+        else:
+            NEW.append(create)
+    else:
+        NEW.append(host.target)
+
+
+def add_hosts_all_subnets(parsed_args):
+    """Get all hosts, without checking the subnet and add them if not in read-only"""
+    for host in CLIENT.hosts():
+        if host.category == "linux":
+            if host.server_id:
+                EXISTING.append(host)
+            if host.server_id is None:
+                create_agentless(host, parsed_args)
+
+
+def add_hosts_specific_subnet(parsed_args, valid_subnets):
```

Comment:
only one parameter is requried here as valid_subnets are always inside of parsed_args